target/linux/lantiq/image/tp-link.mk

```diff
@@ -13,51 +14,51 @@ endef
 
 define Device/tplink_tdw8970
   $(Device/lantiqTpLink)
+  DEVICE_MODEL := TD-W8970
```

Comment:
```
DEVICE_VARIANT := v1
```
Quite important, as future variants use completely different SoC and are unsupported... See also https://openwrt.org/toh/tp-link/td-w8970_v1

Reply:
I personally would also add the v1 to the three subsequent TP-Link devices, since TP-Link always refers to its devices with a version, even if there only is a v1, and you never know whether there will a v2:
https://www.tp-link.com/us/support/download/td-w8980/
https://www.tp-link.com/uk/support/download/archer-vr200/
https://www.tp-link.com/de/support/download/archer-vr200v/

Reply:
Good idea, but only as long as they have an official (explicit) v1.

Reply:
I bet 10 Euros that the have "Ver.1.x" printed on the sticker as any other TP-Link.
And the V1 on the homepage is "explicit". But I won't be mad at you if you omit it here.

Reply:
I've already added the v1.

Reply:
We are generally omitting the minor version of TP-Link devices as they only reflect the software version pre-installed on the devices. For us, only the major version matters and we have for now always denoted them prefixed with a small 'v'.

Reply:
@dangowrt Yes, this is just a misunderstanding. I just wanted to point out that the version is "explicit" in terms of mwarning, i.e. that it's printed on the device. I also support adding it as "v1".